cpp/comms/CMakeLists.txt

```diff
@@ -20,31 +20,50 @@ project(cuML-comms VERSION 0.8.0 LANGUAGES CXX CUDA)
 set(CUML_DIR ${PROJECT_SOURCE_DIR}/.. CACHE STRING "Path to the cuML repo")
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CUML_DIR}/cmake")
 
-find_package(MPI REQUIRED)
-find_package(NCCL)
+option(WITH_UCX "Uses UCX for p2p comms" OFF)
+
+
+find_package(NCCL REQUIRED)
+find_package(CUDA 9.0 REQUIRED)
```

Comment:
I did this because cuML's CMakeLists.txt includes this line: 

```
find_package(CUDA 9.0 REQUIRED)
```

as well as 

```
project(CUML VERSION 0.9.0 LANGUAGES CXX CUDA)
```

I will remove it if it's not needed